src/plugins/chart_expressions/expression_xy/public/components/tooltip/tooltip.scss

```diff
@@ -1,10 +1,11 @@
 .detailedTooltip {
-  @include euiToolTipStyle('s');
   pointer-events: none;
-  max-width: $euiSizeXL * 10;
-  overflow: hidden;
-  padding: $euiSizeS;
-
+  @include euiToolTipStyle('s');
+  & { // stylelint-disable-line no-duplicate-selectors
```

Comment:
This disabled lint rule was added because currently our style doesn't support this deprecation resolution. I've checked with @jbudz and this is ok since we don't want to disable that rule everywhere.